src/client/imagegen.h

```diff
@@ -35,42 +37,40 @@ class SourceImageCache {
 
 	video::IImage* get(const std::string &name);
 
-	// Primarily fetches from cache, secondarily tries to read from filesystem
+	// Primarily fetches from cache, secondarily tries to read from filesystem.
 	video::IImage *getOrLoad(const std::string &name);
 private:
 	std::map<std::string, video::IImage*> m_images;
 };
 
-/*
- * Generates and caches images.
- * The image name defines the image by filename and texture modifiers.
-*/
+// Generates images using texture modifiers, and caches source images.
 struct ImageGen {
-	/*! Generates an image from a full string like
-	 * "stone.png^mineral_coal.png^[crack:1:0".
-	 * The returned Image should be dropped.
-	 * source_image_names is important to determine when to flush the image from a cache (dynamic media)
-	 */
+	// Generates an image from a full string like "stone.png^mineral_coal.png^[crack:1:0".
+	// The returned Image should be dropped.
+	// source_image_names stores all source images which have been used to generate the image.
+	// It is important to determine when to flush the image from a cache (dynamic media).
```

Comment:
if you edit the comments anyway consider documenting the functions according to doxygen

Reply:
I'm not too familiar with doxygen and I only edited a few comments.
Also, it looks like it is not used at many places. So for this PR I will not change it.